.github/workflows/coverage.yaml

```diff
@@ -7,6 +7,7 @@ on:
       - main
   pull_request:
   workflow_dispatch:
+  merge_group:
```

Comment:
We don't need coverage and deploy on merge-groups, only the jobs that are part of the "required status checks".

Reply:
Yes, but I'd prefer to have them there as those could also be broken by incompatible changes. Due to the `if:` condition on the deploy workflow, only the building steps actually run, not the deploying steps.

Reply:
Right, I'll create a PR to be able mark more checks as required.